### PR TITLE
added restriction for editing projects

### PR DESCRIPTION
**Pull request by ahodge10**
Complete functionality for editing projects based on ownership of project and Admin role, but code is dependent on the approval of implementation from other user stories.

### CODE REVIEW

**Review by jurczakn — CHANGES_REQUESTED** [review]
Add a test for the display of the update project button.

**Comment by ahodge10** [comment]
Changes addressed. Merge conflict resolved.

**Comment by ahodge10** [comment]
conflicts resolved

File: src/app/components/pages/project/project-grid-page/project-grid-page.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
                 <h1 *ngIf="project">{{ project.name }}</h1>
                 <div *ngIf="!project">
                     <app-project-welcome-page></app-project-welcome-page>
-                </div>
+                </div> -->
```
Comment (jurczakn) [inline src/app/components/pages/project/project-grid-page/project-grid-page.component.html]: Remove leftover end comment

**Review by jurczakn — CHANGES_REQUESTED** [review]

**Review by jurczakn — APPROVED** [review]
ok

**Pull request merged**